### PR TITLE
DDF-1273:reorganized Admin docs.

### PR DESCRIPTION
**Pull request by ricklarsen**
review:

@oscaritoro 
@stustison 
@ahoffer 
@djblue 
@clockard 
@vinamartin 
@gordocanchola

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/441" target="_blank">ReviewNinja

**Comment by vinamartin** [comment]
:+1:

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -1,99 +1,83 @@
...
+|Configuration Admin JMX Bridge
+|`ddf.ui.admin.api`
+|List of Configuration Filters
 
-== Verify
+==== Defining Platform Settings
 
-. Verify the appropriate features for the DDF application have been installed using the
-`features:list` command to view the KAR file's features.
+The platform settings can be configured by a file located in `DDF_HOME/etc/ddf.platform.config.cfg`
```
Comment (clockard) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: This file no longer exists. It has been replaced with the system.properites file

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 2]
```diff
@@ -1,99 +1,83 @@
...
 
-. Verify that the bundles within the installed features are in an active state.
+The settings in this file can be changed at any time during the {branding} lifecycle (before installation, after, or during run-time) and will immediately update the corresponding properties for the platform global configuration.
```
Comment (clockard) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: Again no platform global configuration anymore. Changes to the system.properites file requires a system restart.

File: platform/admin/docs/src/main/resources/ManContents.adoc (hunk 1 above)
Comment (ricklarsen) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: fixed.

File: platform/admin/docs/src/main/resources/ManContents.adoc (hunk 2 above)
Comment (ricklarsen) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: fixed.

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 3]
```diff
@@ -1,99 +1,84 @@
...
-
-Example:
-    ddf@local>features:addurl mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-
-    ddf@local>features:install catalog-app
-----
+=== Administrative User Interface
 
-== Upgrade 
+The Admin UI is the centralized location for administering the system.
```
Comment (stustison) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: We should be calling this "Admin Console" everywhere and not Admin UI.

Comment (ricklarsen) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: will fix.

**Comment by oscaritoro** [comment]
:+1:

File: platform/admin/docs/src/main/resources/ManContents.adoc (hunk 3 above)
Comment (stustison) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: It probably wouldn't hurt to also provide the default address to where the Admin Console is (https://localhost:8993/admin)

Comment (ricklarsen) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: fair enough. added.

**Comment by oscaritoro** [comment]
Still builds and looks good to me :+1:

**Comment by clockard** [comment]
LGTM :+1:

**Pull request merged**